src/Core.js

```diff
@@ -78,94 +82,106 @@ module.exports = {
 		return true
 	},
 
-	addEntity: function (authorization, body, uri) {
-		return rp({
-			method: 'POST',
-			uri: uri,
-			body: body,
-			headers: {
-				Authorization: authorization,
-				'x-imt-apps-sdk-version': Meta.version
-			},
-			json: true
-		}).catch(err => {
-			vscode.window.showErrorMessage(err.error.message)
-		})
+	addEntity: async function (authorization, body, url) {
```

Comment:
it would be nice to unify argument names to use either `value` or `body`


```suggestion
	addEntity: async function (authorization, value, url) {
```

Reply:
Discussed offline. Closing.